ui-components/src/locales/ar.json

```diff
@@ -704,7 +704,7 @@
         "bn": "বাংলা"
     },
     "leasingAgent": {
-        "contact": "اتصل بوكيل التأجير",
+        "contact": "شركة إدارة",
```

Comment:
All of these should go in `sites/public/page_content/locale_overrides` so that our src/locales are in sync with the general.json, which theoretically should only have additions (not changes) as compared to bloom-housing.

Reply:
Thanks for pointing to `locale_overrides` - I hadn't seen that! I added all of the overrides there instead, and kept all the pre-existing values in `ui-components/src/locales` intact.

I did add a new field to each of the `ui-components/src/locales` files: "website". I think that's the right thing to do (since I'm adding a reference to that field in `ui-components`), rather than only defining "website" in the `locale_overrides` but let me know if you disagree?